app/controllers/contact_controller.rb

```diff
@@ -126,4 +127,10 @@ def referrer_attribute
   def technical_attributes
     { user_agent: request.user_agent }
   end
+
+  def add_cors_header
+    if request.headers["origin"] && request.headers["origin"].end_with?(".gov.uk")
```

Comment:
Small thing, but you could DRY this up by putting `request.headers["origin"]` into a variable:

```
origin_header = request.headers["origin"]

if origin_header && origin_header.end_with?(".gov.uk")
  response.headers["Access-Control-Allow-Origin"] = origin_header
end
```

Reply:
Thanks @Rosa-Fox - i'll change this then merge this PR 👍